modules/gapi/src/backends/fluid/gfluidcore.cpp

```diff
@@ -270,17 +514,40 @@ GAPI_FLUID_KERNEL(GFluidDiv, cv::gapi::core::GDiv, false)
     }
 };
 
+template<typename DST, typename SRC1, typename SRC2>
+static void run_absdiff(Buffer &dst, const View &src1, const View &src2)
+{
+    static_assert(std::is_same<SRC1, SRC2>::value, "wrong types");
+    static_assert(std::is_same<SRC1, DST>::value, "wrong types");
+
+    const auto *in1 = src1.InLine<SRC1>(0);
+    const auto *in2 = src2.InLine<SRC2>(0);
+    auto *out = dst.OutLine<DST>();
+
+    int width = dst.length();
+    int chan = dst.meta().chan;
+    int length = width * chan;
+
+    int x = 0;
+
+#if CV_SIMD
+        absdiff_simd(in1, in2, out, length, x);
+#endif
+        for (; x < length; ++x)
+            out[x] = absdiff<DST>(in1[x], in2[x]);
+}
```

Comment:
indentation is broken

Reply:
I don't see broken indentation. Please explain?

Reply:
it is broken

Reply:
Done.

Reply:
Changed.